src/sentry/static/sentry/app/views/settings/incidentRules/ruleNameOwnerForm.tsx

```diff
@@ -51,7 +51,6 @@ class RuleNameOwnerForm extends React.PureComponent<Props> {
                       model.setValue('owner', ownerValue);
                     }}
                     filteredTeamIds={userTeamIds}
-                    includeUnassigned
```

Comment:
I think these two forms are the only ones that use the `includeUnassigned` prop so if the end goal is to remove the option completely we should remove the `unassignedOption` in the `<SelectMembers>` component and related typing/props